budget-squirrel/src/components/Header/Header.js

```diff
@@ -9,7 +9,7 @@ export default function Header() {
                 <h2>Budget Squirrel</h2>
             </div>
             <div class="header__section header__section__right">
-                <button class="header__button">
+                <button to="/login" class="header__button">
```

Comment:
This should be an anchor tag like the register page. Also, use the constants in RouteConstants to reference the href.

Reply:
Updated 😄